.github/workflows/deploy-unsigned.yml

```diff
@@ -78,8 +78,6 @@ jobs:
       # TODO: Is this really needed?
       - name: "Remove images locally"
         run: |
-          docker rmi "ghcr.io/${{ github.repository }}/vro-rabbitmq:3"
-
```

Comment:
Unnecessary since the new `add-3rd-party-image.sh` script will be used to push the image once.